spring-cloud-gcp-data-datastore/src/test/java/com/google/cloud/spring/data/datastore/it/DatastoreIntegrationTests.java

```diff
@@ -544,6 +546,17 @@ public void testDelete() {
 		assertThat(this.testEntityRepository.findById(1L)).isNotPresent();
 	}
 
+	@Test
+	public void deleteAllByIdTest() {
+		assertThat(this.testEntityRepository.findAllById(Arrays.asList(1L, 2L))).hasSize(2);
+		// cast to SimpleDatastoreRepository for method be reachable with Spring Boot 2.4
+		SimpleDatastoreRepository simpleRepository = AopTestUtils
```

Comment:
`this.testEntityRepository` is a Spring Proxy object, and could not cast it to `SimpleDatastoreRepository` directly. Found this utility method to unwrap it.

Reply:
oh, this is very cool!